build/deps/textlint.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:lts-alpine
```

Comment:
Let's use a fixed tag with a version. Dependabot should be able to send updates for it.

Reply:
Thank you, fixed